app/views/admins/pages/dashboard.html.erb

```diff
@@ -2,3 +2,4 @@
 
 <p><%= link_to "All Kudos list", admins_kudos_path %></p>
 <p><%= link_to "Employees list", admins_employees_path %></p>
+<p><%= link_to "Company values", admins_company_values_path %></p>
```

Comment:
I would consider the names of the links. IMO Kudos and Employees are enough.

Reply:
I will rename the links as you suggest.